src/main/java/com/iota/iri/network/pipeline/TransactionProcessingPipelineImpl.java

```diff
@@ -57,7 +57,7 @@
 public class TransactionProcessingPipelineImpl implements TransactionProcessingPipeline {
 
     private static final Logger log = LoggerFactory.getLogger(TransactionProcessingPipelineImpl.class);
-    private ExecutorService stagesThreadPool = Executors.newFixedThreadPool(6);
+    private ExecutorService stagesThreadPool = Executors.newFixedThreadPool(7);
```

Comment:
Can you please either:
1. create an issue to add a reflective call like `getSubClasses(Stage.class).size()`
2. Add it now if you have time (better not add another dependency if you can help it).

Reason:
I would have for sure missed this in my review if you would have forgotten to change

Reply:
The easiest way to do this would be to use the reflections class so I'm going to create the issue for it and draft a new PR specifically for that.